### PR TITLE
Disableable double step

### PR DESCRIPTION
**Pull request by AdrieanKhisbe**
Can I disablez button pleases? 😃 

(also fix storybook broserlist webpack kamoulox)

Capture d’écran 2019-11-12 à 16 18 50

### CODE REVIEW

File: packages/@coorpacademy-components/.storybook/webpack.config.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ module.exports = ({config}) => {
             '@babel/preset-env',
             {
               targets: {
-                overrideBrowserslist: ['last 2 versions']
```
Comment (silouone) [inline packages/@coorpacademy-components/.storybook/webpack.config.js]: 👍 soz encore ;'(

File: packages/@coorpacademy-components/src/atom/input-doublestep/test/fixtures/disabled.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+export default {
+  props: {
+    confirmValue: 'Whatever',
+    type: 'doublestep',
+    toggleValue: 'Delete Platform',
+    disabled: true,
+    cancelValue: 'Whatever also',
+    description:
+      'Are you sure you want to delete your platform? Deleting your platform is irreversible!',
+    onClick: () => console.log('Sorry, wont happen')
```
Comment (silouone) [inline packages/@coorpacademy-components/src/atom/input-doublestep/test/fixtures/disabled.js]: 😂

File: packages/@coorpacademy-components/.storybook/webpack.config.js (hunk 1 above)
Comment (AdrieanKhisbe) [reply packages/@coorpacademy-components/.storybook/webpack.config.js]: nopro :)

File: packages/@coorpacademy-components/src/atom/input-doublestep/test/fixtures/disabled.js (hunk 2 above)
Comment (AdrieanKhisbe) [reply packages/@coorpacademy-components/src/atom/input-doublestep/test/fixtures/disabled.js]: j'aurais pu whatever la description aussi... 🤷‍♂

**Comment by AdrieanKhisbe** [comment]
```
Successfully published:
 - @coorpacademy/components@9.5.1
 - @coorpacademy/player-web@3.3.1
```

**Pull request merged**